app/javascript/dashboard/routes/dashboard/helpcenter/components/ArticleSearch/ArticleSearchResultItem.vue

```diff
@@ -92,13 +74,16 @@ export default {
     },
   },
   methods: {
-    handleInsert() {
+    handleInsert(e) {
+      e.stopPropagation();
```

Comment:
Why do we need to stopPropagation here? It shows an underlying implementation problem. Unless, it is absolutely required, let us not do this.

Reply:
The parent item is a button. We need to use stop Propagation so the event won't go up.